docs/contributing.md

```diff
@@ -0,0 +1,136 @@
+# Contributing
+
+This document describes how to install KubeSaw in a development environment.
+
+## Prerequisites
+
+### OpenShift Cluster
+Ensure you have access to an OpenShift 4.6+ cluster with cluster admin privileges and log in using `oc login`
+
+### Required Tools
+Install the [required tools](required-tools.md).
+
+### Authentication
+Configure authentication for the cluster using one of the following options:
+
+1. Contact a member of the Dev Sandbox Team for instructions on how to configure the cluster to use our internal Dev SSO.
```

Comment:
Maybe
```suggestion
1. Contact a member of the KubeSaw Team for instructions on how to configure the cluster to use our internal Dev SSO.
```
???

Reply:
yeah I was in doubt here, but makes sense, just for consistency. thanks!